src/components/Good.tsx

```diff
@@ -0,0 +1,12 @@
+type GoodProps = {
+  goodName: string;
+  key: number;
```

Comment:
The `key` prop should not be included in the props type definition for a component, as `key` is not a prop that gets passed to the component function; it's handled by React itself. Remove the `key` prop from the `GoodProps` type.